moveit_experimental/moveit_jog_arm/src/jog_ros_interface.cpp

```diff
@@ -90,6 +90,11 @@ JogROSInterface::JogROSInterface()
   ros::Subscriber joints_sub =
       nh.subscribe(ros_parameters_.joint_topic, 1, &JogInterfaceBase::jointsCB, dynamic_cast<JogInterfaceBase*>(this));
 
+  // ROS Server for allowing drift in some dimensions
+  ros::ServiceServer drift_dimensions_server =
+      nh.advertiseService(nh.getNamespace() + "/" + ros::this_node::getName() + "/change_drift_dimensions",
```

Comment:
This should be documented, ideally with an example client. I would suggest putting the topic name in a constant and if possible adding a simple test that verifies this works.

Reply:
OK. There's some documentation in the message type here, but I'll add a test too.

> # For use with moveit_jog_arm Cartesian planner
> #
> # Allow the robot to drift along these dimensions in a smooth but unregulated way.
> # Give 'true' to enable drift in the direction, 'false' to disable.
> # For example, may allow wrist rotation by drift_x_rotation == true.

Reply:
done